configs/dictionaries/uxmpracticetest_percentages.yaml

```diff
@@ -0,0 +1,4 @@
+---
+A: 50
```

Comment:
If you're doing this for a real test, I tend to ship it with the current version set to 100% and any variants set to 0. (It also helps to set a short cookie time while doing this bit)
This approach gives you a bit of time to check that cookies are set correctly and just kick the tyres. You can test the other variants with the GOV.UK chrome extension at this point.

Once you're confident everything is OK, you can start upping the variant percentages and the cookie age.

It's probably also important to know that because the test adds a `Vary` header for your test, it invalidates Fastly caches for the affected pages when that header starts to appear. That's probably not super important for this page at the moment, but if you happen to be doing a test on a mega-popular page that has a lot of backend work, it can be a factor.